src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Suppliers/add.html.twig

```diff
@@ -40,4 +40,7 @@
   {{ parent() }}
 
   <script src="{{ asset('themes/new-theme/public/supplier_form.bundle.js') }}"></script>
+  <script src="{{ asset('../js/tiny_mce/tiny_mce.js') }}"></script>
```

Comment:
Mmm, seems like you copied this from the product page Then it's not useful any more, this page is old But on recent one just including the TinyMceEditor component should be enough This simplifies the init of the page and it will make it easier if we want to change the editor library some day